dev/com.ibm.ws.concurrent.mp.1.0/resources/com/ibm/ws/concurrent/mp/resources/CWWKCMessages.nlsprops

```diff
@@ -0,0 +1,53 @@
+###############################################################################
+# Copyright (c) 2018 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#CMVCPATHNAME com.ibm.ws.concurrent/resources/com/ibm/ws/concurrent/resources/CWWKCMessages.nlsprops
+#ISMESSAGEFILE TRUE
+#NLS_ENCODING=UNICODE
+#
+#COMPONENTPREFIX CWWKC
+#COMPONENTNAMEFOR CWWKC MicroProfile Concurrency
+#
+# NLS_MESSAGEFORMAT_VAR
+#
+#   Strings in this file which contain replacement variables are processed by the MessageFormat 
+#   class (single quote must be coded as 2 consecutive single quotes ''). Strings in this file 
+#   which do NOT contain replacement variables are NOT processed by the MessageFormat class 
+#   (single quote must be coded as one single quote '). 
+
+# All messages must use the range CWWCK1150 to CWWCK1199 except those specifically identified as moved
+
+CWWKC1150.duplicate.context=CWWKC1150E: The same thread context type, {0}, is provided by multiple thread context providers which are available to the application: {1}, {2}. 
+CWWKC1150.duplicate.context.explanation=The MicroProfile Concurrency specification permits at most a single thread context provider for each context type.
+CWWKC1150.duplicate.context.useraction=Update the application and/or the libraries that are available to it such that there is only a single thread context provider of the specified type.
+
+CWWKC1151.context.lists.overlap=CWWKC1151E: The following thread context types are configured to be both cleared and propagated: {0}.
+CWWKC1151.context.lists.overlap.explanation=A thread context type can be configured in at most one category when building or injecting a ManagedExecutor.
+CWWKC1151.context.lists.overlap.useraction=Review and correct the categorization of thread context types in your ManagedExecutor builder or ManagedExecutorConfig annotation.
+
+CWWKC1152.context.lists.overlap=CWWKC1152E: The following thread context types are configured in more than one category of (cleared, propagated, unchanged): {0}.
+CWWKC1152.context.lists.overlap.explanation=A thread context type can be configured in at most one category when building or injecting a ManagedExecutor.
```

Comment:
Should this ben ThreadContext rather than ManagedExecutor?

Reply:
Yes, I added a commit to fix it.